modin/core/dataframe/algebra/default2pandas/groupby.py

```diff
@@ -217,12 +217,12 @@ def fn(
             grp = df.groupby(by, axis=axis, **groupby_kwargs)
             func = cls.get_func(agg_func, **kwargs)
             result = func(grp, *agg_args, **agg_kwargs)
+            method = kwargs.get("method")
 
             if isinstance(result, pandas.Series):
-                result = result.to_frame()
+                result = result.to_frame(method or MODIN_UNNAMED_SERIES_LABEL)
```

Comment:
So `method` can be used as name for `to_frame`. Is it a mistake or not?

Reply:
well, it was not an actual mistake, but indeed, the logic was based on a hacky assumption that the 'method' argument could only be a `"size"` (for other methods we always pass `None`) so the logic would break once we decide to actually pass `method` argument for every groupby method.

So thank you for the catch! I've moved "size" renaming logic to a `BaseQueryCompiler.groupby_size` method (so it's now aligned with pandas qc).